thorlcr/slave/slavmain.cpp

```diff
@@ -1939,7 +1939,9 @@ class CJobListener : public CSimpleInterface
                         CJobSlave *job = jobs.find(jobKey.get());
                         if (job)
                         {
-                            if (job->getOptBool("dumpInfoOnAbort"))
+                            bool dumpInfo;
+                            msg.read(dumpInfo);
```

Comment:
Could there be a compatibility issue ?  Should we check for dumpInfoOnAbort option before always reading it ?

Reply:
No because this is a Thor master->slave message in same cluster only, and they must always be on same build.